src/main/java/com/game/Battleship/repository/BoardRepository.java

```diff
@@ -0,0 +1,66 @@
+package com.game.Battleship.repository;
+import com.game.Battleship.models.*;
+import org.springframework.stereotype.Service;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Service
+public class BoardRepository {
+
+    private final PlayerBoard playerBoard;
+    private final ComputerBoard computerBoard;
+    private final List<Board> boardList;
+
+    public BoardRepository(){
+        this.playerBoard = new PlayerBoard();
+        this.computerBoard = new ComputerBoard();
+        this.boardList = new ArrayList<>(Arrays.asList(
+                this.playerBoard,
+                this.computerBoard
+        ));
+
+    }
+
+    public List<Board> allBoards(){
+        return boardList;
+    }
```

Comment:
Your extra lines are not consistent in this file.  See lines 22, 27, 39, 56, 63. Pick a style and try to stick with it.